src/NuGet.Clients/PackageManagement.VisualStudio/Utility/LegacyCSProjPackageReference.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Frameworks;
+using System;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    /// <summary>
+    /// A data transfer type for calls to <see cref="EnvDTEProjectAdapter" />.
+    /// </summary>
+    public class LegacyCSProjPackageReference
+    {
```

Comment:
ideally immutable type, so { get; }

---
In reply to: [84563664](https://github.com/NuGet/NuGet.Client/pull/950#discussion_r84563664) [](ancestors = 84563664)